tests/test_shortcuts.py

```diff
@@ -91,7 +91,7 @@ def gen(num):
         self.assertEqual(list(iter_chunks("01234", 2)), ["01", "23", "4"])
         self.assertEqual(list(iter_chunks("012345", 2)), ["01", "23", "45"])
 
-        file_obj = open("chunks_file", "r")
+        file_obj = open(os.path.dirname(__file__) + "/chunks_file", "r")
```

Comment:
This is actually breaking the test for me - well, it works or fails depending on the working directory while run.
Currently, it fails while run by tox or by "make test".

It fails if the working directory is the directory of the test:

```
  $ python test_shortcuts.py 
  .......E........
  ======================================================================
  ERROR: test_iter_chunks (__main__.TestShortcuts)
  ----------------------------------------------------------------------
  Traceback (most recent call last):
    File "test_shortcuts.py", line 94, in test_iter_chunks
      file_obj = open(os.path.dirname(__file__) + "/chunks_file", "r")
  IOError: [Errno 2] No such file or directory: '/chunks_file'
  
  ----------------------------------------------------------------------
  Ran 16 tests in 0.703s
  
  FAILED (errors=1)
```

Think you could fix that?  (os.path.dirname(__file__) or '.') would do the trick.